src/styles/card.scss

```diff
@@ -0,0 +1,127 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  gap: 30px 48px;
+
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    max-width: 200px;
```

Comment:
The 'max-width' property is unnecessary here since the grid column width is already set to 200px, which will constrain the card width. You can safely remove this line.